Explorer/Assets/DCL/Rendering/Reflections/ReflectionProbeRenderer.cs

```diff
@@ -0,0 +1,28 @@
+using UnityEngine;
+
+public class ReflectionProbeRenderer : MonoBehaviour
+{
+    public float IntervalInSeconds = 5f;
+    private ReflectionProbe reflectionProbe;
```

Comment:
this could also be a `[SerializeField] private` so you can reference the component from the prefab instead of doing a `GetComponent` on `Start()`

Reply:
done